DOCS/canonical-identity-platform-how-to-integrate-with-oidc-compatible-charms.md

```diff
@@ -2,13 +2,12 @@ The Identity Platform provides seamless integration with your OIDC compatible ch
 1. You have deployed the [identity platform bundle](link to tutorial).
 2. You have deployed an OIDC compatible charmed application.
 
-To integrate you need to run:
+To connect an OIDC compatible charmed application with the identity-platform bundle, integrate it with `hydra`:
```

Comment:
Here and everywhere else (I've noticed a few more places like this): Code names should be given in code formatting. E.g., here, `identity-platform`.